app/palettes.ts

```diff
@@ -165,6 +165,7 @@ type SteppedPalette<T> = Omit<Palette<T>, "interpolator"> & {
   colors: ReadonlyArray<string>;
 };
 const steppedPaletteSteps = [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1];
+// Diverging and sequential palette keys must correspond to types in configurator-types.ts!
```

Comment:
`const divergingPaletteKeys: DivergingPaletteType[] = [` should error if the types are not synced in configurator-types. This would remove the need for the comment ;)

Reply:
🤦 👍